azure-pipelines.yml

```diff
@@ -85,7 +85,7 @@ stages:
     - job: 'MacOSWheels'
       timeoutInMinutes: 60
       pool:
-        vmImage: 'macOS-12'
+        vmImage: 'macOS-13'
```

Comment:
I didn't check the toolchain versions on MacOS-12 and if it's worth testing a bit longer, but it's LGTM.